### PR TITLE
Fix/Add comments on cadvisor implementations

### PR DESCRIPTION
**Pull request by alculquicondor**
**What type of PR is this?**

/kind cleanup
/kind documentation

**What this PR does / why we need it**:

* Fix a misleading comment in cadvisor_linux.
* Add an explanation on why a parameter is not being used in cadvisor_windows, according to some reading of the code (need validation).
* Fix lint errors in `pkg/kubelet/cadvisor` and `pkg/kubelet/cadvisor/testing` (#68026)

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/cadvisor/cadvisor_windows.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ func (cu *cadvisorClient) ContainerInfo(name string, req *cadvisorapi.ContainerI
 }
 
 func (cu *cadvisorClient) ContainerInfoV2(name string, options cadvisorapiv2.RequestOptions) (map[string]cadvisorapiv2.ContainerInfo, error) {
+	// This method is only expected to be used for the entire node. Then, all the containers' info should be retrieved.
```
Comment (neolit123) [inline pkg/kubelet/cadvisor/cadvisor_windows.go]: might as well move this as a export comment on to of the function:
e.g.
```
ContainerInfoV2 is only .....
```

**Review by neolit123 — COMMENTED** [review]
thanks for the PR @alculquicondor 
added one minor comment.

/priority backlog
/ok-to-test

**Comment by alculquicondor** [comment]
I fixed lint errors to make the PR more relevant.

**Comment by alculquicondor** [comment]
/assign @vishh 

for hack/.golint_failures

**Comment by oomichi** [comment]
@alculquicondor Unfortunately this seems necessary to be rebased for the latest master.
Could you do that?

**Pull request merged**